client/src/pages/ProfileSetupPage/ProfileSetupPage.js

```diff
@@ -0,0 +1,182 @@
+// importing modules
+import React, { Component } from 'react';
+import { Box, Container, Typography } from '@material-ui/core';
+import momentTZ from 'moment-timezone';
+
+// importing components
+import ProfileStep1 from './ProfileSteps/ProfileStep1/ProfileStep1';
+import ProfileStep2 from './ProfileSteps/ProfileStep2/ProfileStep2';
+import ProfileStep3 from './ProfileSteps/ProfileStep3/ProfileStep3';
+
+class ProfileSetupPage extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            step: 1,
+            username: '',
+            timezoneName: momentTZ.tz.guess(true),
+            timeAvlFrom: '', 
+            timeAvlUntil: '',
+            daysAvl: {
+                sunday: false,
+                monday: false,
+                tuesday: false,
+                wednesday: false,
+                thursday: false,
+                friday: false,
+                saturday: false
+            },
```

Comment:
If these are defaults should we have monday-friday selected true unless deselected?